seacrowd/sea_datasets/mc4_indo/mc4_indo.py

```diff
@@ -0,0 +1,146 @@
+import gzip
+import json
+from typing import List
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+
+_DATASETNAME = "mc4_indo"
+_DESCRIPTION = """\
+    A thoroughly cleaned version of the Indonesia split of the multilingual colossal, cleaned version of Common Crawl's web crawl corpus (mC4). This portion represents the Indonesian language content that has been extracted and processed from the larger mC4 dataset. The extraction and cleaning process was conducted by AllenAI and resulted in a curated collection of Indonesian language data. For more information about the original mC4 dataset and its preparation, please refer to the source hosted at the address https://huggingface.co/datasets/allenai/c4.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/indonesian-nlp/mc4-id"
+_LICENSE = Licenses.ODC_BY.value
+_LOCAL = False
+_LANGUAGES = ["ceb"]
+_DATASETNAME = "cebuaner"
```

Comment:
Wrong _LANGUAGES and _DATASETNAME, please fix

Reply:
so sorry, completely missed this :(